containers/Dill.ts

```diff
@@ -48,7 +48,7 @@ export function useDill(): UseDillOutput {
         const dillContract = dill.attach(DILL);
         const feeDistributorContract = feeDistributor.attach(FEE_DISTRIBUTOR);
         const distributionEventsRaw = await feeDistributor.queryFilter(
-          feeDistributor.filters.CheckpointToken(),
+          feeDistributor.filters.CheckpointToken(null, null),
```

Comment:
Debugging this type prompted the PR. Passing explicit `null`s has the same effect for now, it will be resolved [soon](https://github.com/ethereum-ts/TypeChain/issues/379).